controllers/fg_handler.go

```
@@ -78,27 +78,18 @@ func (r *KataConfigOpenShiftReconciler) processFeatureGates() error {
 
 	// Check which feature gates are enabled in the FG ConfigMap and
 	// perform the necessary actions
-
-	if IsEnabled(fgStatus, TimeTravelFeatureGate) {
-		r.Log.Info("Feature gate is enabled", "featuregate", TimeTravelFeatureGate)
-		// Perform the necessary actions
-		r.handleTimeTravelFeature(Enabled)
-	} else {
-		r.Log.Info("Feature gate is disabled", "featuregate", TimeTravelFeatureGate)
-		// Perform the necessary actions
-		r.handleTimeTravelFeature(Disabled)
+	if r.kataConfig.Spec.EnablePeerPods {
+		if IsEnabled(fgStatus, ConfidentialFeatureGate) {
+			r.Log.Info("Feature gate is enabled", "featuregate", ConfidentialFeatureGate)
+			// Perform the necessary actions
+			r.toggleFeatureConfidential(true)
```

`toggleFeatureConfidential()` can fail. Ignoring the error seems wrong.